app/workers/company_worker.rb

```diff
@@ -0,0 +1,24 @@
+class CompanyWorker
+  include Sidekiq::Worker
+
+  def perform(company, count)
```

Comment:
Unused method argument - `count`. If it's necessary, use `_` or `_count` as an argument name to indicate that it won't be used.<br>Assignment Branch Condition size for perform is too high. [29.17/15]